test/fixtures/rules/std_rule_cases/L016.yml

```diff
@@ -4,7 +4,7 @@ test_pass_line_too_long_config_override:
   # Long lines (with config override)
   pass_str: "SELECT COUNT(*) FROM tbl\n"
   configs:
-    rules:
+    indentation:
```

Comment:
`indentation` feels like a strange place for `max_line_length`. Should we consider a more inclusive term such as `formatting`?

Reply:
good point 🤔 . Rather than making a new section for it - I wonder whether I should just bring it up to the parent `sqlfluff` section. 👍